webcompat/webhooks/__init__.py

```diff
@@ -39,11 +39,19 @@ def hooklistener():
             issue_body = payload.get('issue')['body']
             issue_title = payload.get('issue')['title']
             issue_number = payload.get('issue')['number']
+            issue_status = payload.get('issue')['status']
+            issue_creation_time = payload.get('issue')['created_at']
+            issue_last_change_time = payload.get('issue')['updated_at']
+            if 'reported_from' in payload.get('issue'):
+                issue_reported_from = payload.get('issue')['reported_from']
+            else:
+                issue_reported_from = 'null'
```

Comment:
instead of if/else here. You could set issue_reported_from = 'null' at the beginning after line 37 (aka a default value).
Then just setting the value if it's found in the payload.